package.json

```diff
@@ -59,6 +58,7 @@
     "ember-cli-typescript": "^4.2.1"
   },
   "devDependencies": {
+    "@ember/render-modifiers": "^1.0.2",
```

Comment:
quite sure this needs to be a dependency? we use this on the contenteditable class

Reply:
oh I didnt realize we already had it so I just `ember install`ed it.
Bit annoying it then overwrites your package.json for no reason